node_modules/oae-core/manageaccess/js/manageaccess.js

```diff
@@ -29,6 +29,9 @@ define(['jquery', 'oae.core', 'jquery.autosuggest'], function($, oae) {
 
         // Caches the member updates that need to be applied
         var membersUpdates = {};
+        
+        // Caches members pending addition
+        var pendingMembers = {};
```

Comment:
Sure. I'll update the docs. I originally implemented it as an array, but handling all the possible adds and removes got a little messy.